src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -1028,7 +1025,6 @@ void gfx_draw_string_with_y_offsets(
     std::memcpy(text_palette, info.palette, sizeof(info.palette));
 
     gCurrentFontSpriteBase = info.font_sprite_base;
-    gCurrentFontFlags = info.flags;
```

Comment:
Wouldn't it get new values here? Or is `info.flags` never modified? Because if it's the latter, then we can also delete the `flags`

Reply:
Earlier on it would do ```info.flags = gCurrentFontFlags``` and info.flags is used in some cases.

Reply:
To me, it seems like it basically saves whatever info.flags used to be before. But since the only thing that actually seems to vary in `info.flags` is whether or not `TEXT_DRAW_FLAG_TTF` is set, and we don't want that to stick anyway, it seems like nothing useful is written to `gCurrentFontFlags`.

Did you make sure to test with English/German (for the sprite font) and a TTF language (like Korean) to see if it did not break anything?

Reply:
Everything seems to be working as expected. Given I eliminated all references and the only write was info.flags or 0 I already expected that nothing changes.